E3Next/Utility/e3util.cs

```diff
@@ -494,8 +494,11 @@ public static void BuffInfoToDictonary(string s, Dictionary<Int32, Int64> list,
                     _buffInfoTempList.Clear();
                     string tstring = s.Substring(start, end - start);
                     StringsToNumbers(tstring, ',', _buffInfoTempList);
-                    result.Add((Int32)_buffInfoTempList[0], _buffInfoTempList[1]);
-
+                    if (result.ContainsKey((Int32)_buffInfoTempList[0])) {
```

Comment:
this can be simplified thusly `result[(int)_buffInfoTempList[0]] = _buffInfoTempList[1];`

this is safe to do, as the `key exists` check is built in. if the key does not exist, it will be added, else it will be updated

Reply:
I'll change it, but it also might be nice to get the Max() of `result[(Int32)_buffInfoTempList[0]] = Math.Max(result[(Int32)_buffInfoTempList[0]], _buffInfoTempList[1]);` in, so that duplicate entries we take the longest duration for

Reply:
Oh yeah the max is fine, I was just meaning to use the shorthand syntax